locales/en/messages.json

```diff
@@ -2805,7 +2805,7 @@
         "message": "Sat ID"
     },
     "gpsSignalStatusQly": {
-        "message": "Status / Quality"
+        "message": "Status &nbsp;&nbsp;&nbsp; | &nbsp;Quality"
```

Comment:
Sorry, I arrive too late to this, I was out... this is not the way to do that. Using "spaces" to align elements will break the alignment with the translations.

Reply:
you are right, this was forgotten in the battle.
I think I change in a new PR, not to confuse with this one